Game/Config/DefaultEngine.ini

```diff
@@ -39,6 +39,9 @@ bAllowClientSideNavigation=True
 TargetedHardwareClass=Desktop
 DefaultGraphicsPerformance=Maximum
 
+[/Script/OnlineSubsystemUtils.IpNetDriver]
+;ReplicationDriverClassName=/Script/ReplicationGraph.BasicReplicationGraph
```

Comment:
why is this required?

Reply:
So this is used to set which replication class implementation you want to use. I've commented it out, as I don't want to enable it by default (not currently anyway).